network-mock/src/lib.rs

```diff
@@ -178,7 +178,7 @@ pub mod tests {
         type Item = TestRecord;
 
         const BUFFER_SIZE: usize = 8;
-        const NAME: &'static str = "hello_world";
+        const NAME: &'static str = "test-wasm";
```

Comment:
"test-wasm" is the topic name for the wasm tests. Did you mean "test-mock" here?